src/Components/Blazor/Blazor/src/Rendering/WebAssemblyRenderer.cs

```diff
@@ -32,7 +32,10 @@ public WebAssemblyRenderer(IServiceProvider serviceProvider): base(serviceProvid
         }
 
         internal void DispatchBrowserEvent(int componentId, int eventHandlerId, UIEventArgs eventArgs)
-            => DispatchEvent(componentId, eventHandlerId, eventArgs);
+        {
+            // Event dispatching from the browser is fire-and-forget
+            _ = DispatchEventAsync(componentId, eventHandlerId, eventArgs);
```

Comment:
I expect things like error handling to happen at this layer, so either the Renderer does it (on the base class) or we have some code here to at least log to the console or similar (equivalent to what we used to do)

Reply:
The error handling is inside `DispatchEventAsync`